### ISSUE TITLE
Strip AVR32 libraries to fix Hudson problem

### ISSUE THREAD

**Issue by kblomqvist** [jsnyder/avr32-toolchain#21]
What's this all about?
http://www.avrfreaks.net/index.php?name=PNphpBB2&file=viewtopic&t=113202

This can be fixed by calling avr32-strip from Makefile. I have also made a python script to fix this:
https://gist.github.com/kblomqvist/4100575#file-fixhudson-py

**Comment by jsnyder** [comment]
It seems like it should be fairly trivial to add the fix to the makefile.  The only reason I'd not just pull in the Python script is that it's not currently on the dependency list.

I see the suggestion to use the find command on that forum, although I'd want to check that that works for both gnu and bsd variants of that tool.

Does this just crop up during debugging?  I'm busy with quite a bit of other stuff at the moment and I'm more actively developing on ARM platforms at the moment until I finish up a dissertation :-)

**Comment by kblomqvist** [comment]
> It seems like it should be fairly trivial to add the fix to the makefile. The only reason I'd not just pull in the Python script is that it's not currently on the dependency list.

Yep, it should be integrated into Makefile. It wasn't even in my intention to use the Python script to fix this (that was only an example).

> Does this just crop up during debugging?

Yes.

> I'm busy with quite a bit of other stuff at the moment and I'm more actively developing on ARM platforms at the moment until I finish up a dissertation :-)

That's all fine :) I was planning to fix this issue by myself. Opening the issue was more like a note for me.
